File: php/getting-latest-articles/GetAll.php
// DocSection: latest_articles_get_all
<?php
// Tip: Find more about PHP SDKs at https://docs.kontent.ai/php

// Defined by Composer to include required libraries
require __DIR__ . '/vendor/autoload.php';

use Kentico\Kontent\Delivery\DeliveryClient;

$client = new DeliveryClient('975bf280-fd91-488c-994c-2f04416e5ee3');

$items = $client->getItems();
// EndDocSection

File: java/getting-latest-articles/GetAll.java
// DocSection: latest_articles_get_all
// Tip: Find more about Java/JavaRx SDKs at https://docs.kontent.ai/java
import com.github.kentico.kontent.delivery;

DeliveryClient client = new DeliveryClient("975bf280-fd91-488c-994c-2f04416e5ee3");

List<NameValuePair> params = DeliveryParameterBuilder.params()
    .build();

ContentItemsListingResponse listingResponse = client.getItems(params);
// EndDocSection